build/common/Makefile.common.mk

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
+
+FINDFILES=find . \( -path ./.git -o -path ./.github -o -path ./.go \) -prune -o -type f
+XARGS = xargs -0 ${XARGS_FLAGS}
+CLEANXARGS = xargs ${XARGS_FLAGS}
+
+# lint-yaml:
```

Comment:
temporarily disable YAML lint since currently there is no YAML and relate package, will re-enable it later when prototype is merged in